source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -291,6 +308,42 @@ ansible-playbook -i /usr/share/ovirt-engine-metrics/bin/ovirt-engine-hosts-ansib
 5. Use the vNIC profile of this new logical network in a vNIC of the desired VM.
 6. [If requirements are fulfilled, libvirt tries to propagate the MTU into the VM,](#set-the-mtu-by-libvirt). If a subnet is defined for the network, the [MTU is offered via DHCP to the VM](#set-the-mtu-by-dhcp).
 
+#### AutoSync Flow
+
+This is an example flow where the MTU is set by the external provider.
+1. Create a new external logical network with [default MTU](#default-value-for-mtu)
+   on a provider.
+2. If the provider supports the [MTU extenstion][16], it will expose the MTU
+   that is guaranteed to pass through the data path of the segments in the
+   network.
+   During the [automatic synchronization][18] the MTU in the provider's network
+   representation is applied to oVirt Engine's representation.
```

Comment:
"representation" leaves me wondering what you mean
maybe: to the external network on oVirt Engine

Reply:
done